quill-core/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -111,3 +111,11 @@ case class Assignment(input: Ident, property: String, value: Ast)
 //************************************************************
 
 case class Dynamic(tree: Any) extends Ast
+
+case class Binding(key: String, code: String) extends Ast
```

Comment:
Is `code` the string representation to be used by `AstShow`? If yes, wdyt about `toString`?